14/umbraco-cms/tutorials/editors-manual/working-with-content/README.md

```diff
@@ -48,234 +33,276 @@ For ease, there are shortcut keys for certain editor functions. Use the followin
+1. Select the link in the Rich Text Editor.
+   - For text links, click the cursor anywhere within the link text. For an image, click the image itself.
+2. Click the **Remove Link** button which will remove the hyperlink.
+3. Alternatively, you can click the **Insert/Edit Link** button and remove the link from the **Link** field.
 
 ### Working with Images
 
 To display images on a page the images must be uploaded to your Umbraco media library.
 
-Many organisations set up a media library containing images that editors can use on their pages. The procedure for uploading an image varies slightly depending on which method your organisation has adopted, so check with your system administrator what is applicable in your organisation.
+Many organizations set up a media library containing images that editors can use on their pages.  Others allow their editor's free use of their images. The procedure for uploading an image varies slightly depending on which method your organization uses. Check with your system administrator about what applies to your organization.
 
 #### Inserting an Image from the Media Library
 
-1. Place the cursor in the editor where you want to insert your image.
-2. Click the **Media Picker** button on the toolbar. The **Select Media** tree opens on the right-side of the editor.
-3. Select the folder in which your image is stored.
-4. Click the thumbnail of the image. The **Edit selected media** tree opens.
-5. Enter an **Alternative Text** and **Caption** for your image. Both these fields are optional.
-6. Click **Select** to add the image.
+1. Place the cursor in the Rich Text Editor where you want to insert your image.
+2. Click the **Media Picker** button from the toolbar.
+3. Select the folder in which the image is.
+4. Click the thumbnail of your chosen image to open the image properties menu.
+5. Enter a name/description for the image in the **Caption (optional)** field.
+   - It is important to add descriptive titles to images as these are used to assist visually impaired users.
+6. Click **Select**.
 
-![Media Picker](../../../../../10/umbraco-cms/tutorials/editors-manual/working-with-content/images/Media-picker-v9.png)
+![Inserting an Image from the Media Library](images/Inserting-Image-from-the-Media-Library-v11.png)
 
 #### Inserting an Image from your Computer
 
-You can upload images directly from your local directory in the rich text editor. These images will be stored in the Umbraco media library. Ensure that the image is placed in the correct location within the library.
+You can upload images directly from the Rich Text Editor on the page you are editing. These images will be stored in the Umbraco Media Library. Therefore care should be taken to ensure that the image is placed in the correct location within the library. If you click the plus icon underneath the search bar in the media picker slide-out menu you can create folders in the media library.
```

Comment:
```suggestion
You can upload images directly from the Rich Text Editor on the page you are editing. These images will be stored in the Umbraco Media Library. Therefore, it would be best to ensure the image is placed in the correct location within the library. If you click the plus icon underneath the search bar in the media picker slide-out menu you can create folders in the media library.
```